vdsm.spec.in

```diff
@@ -151,11 +151,12 @@ Requires: libvirt-lock-sanlock
 # TODO: require 7.6.0-4+ once available on Centos
 Requires: libvirt-daemon-kvm >= 7.6.0-2
 %else
-# AV 8.5, https://bugzilla.redhat.com/2003644
-Requires: libvirt-daemon-kvm >= 7.6.0-4.module+el8.5.0+12786+c4633d9a
+# Zero-copy migrations, https://bugzilla.redhat.com/2089434
+Requires: libvirt-daemon-kvm >= 8.0.0-5.4.module+el8.6.0+16370+bb85faee
 %endif
 
-Requires: python3-libvirt
+# VIR_MIGRATE_ZEROCOPY flag, https://bugzilla.redhat.com/2089434
+Requires: python3-libvirt >= 8.0.0-1.1.module+el8.6.0+16381+3abc475c
```

Comment:
Is a satisfying version already available in el8stream? I can see:
```
[root@0e3b0712f5c0 /]# dnf info python3-libvirt
Name         : python3-libvirt
Version      : 8.0.0
Release      : 1.module_el8.6.0+1087+b42c8331
```
but not sure if this one will make dnf happy

Reply:
Probably not yet, the corresponding bug (https://bugzilla.redhat.com/2119429) is in VERIFIED status.

Reply:
Unfortunately, support for zero-copy migrations is not available in CentOS Stream 8 libvirt, so I had to disable it there in the last update.